arangod/GeneralServer/GeneralServer.cpp

```diff
@@ -90,11 +134,9 @@ bool GeneralServer::openEndpoint(IoContext& ioContext, Endpoint* endpoint) {
   }
 
   auto task = std::make_shared<GeneralListenTask>(*this, ioContext, endpoint, protocolType);
-  if (!task->start()) {
-    return false;
-  }
+  _listenTasks.emplace_back(task);
```

Comment:
is there more than one listen task ?

Reply:
There is one listen task per server endpoint.
If you make the server listen to multiple endpoints, multiple listen tasks will be created (sequentially).